config/dynamic-scaling-configuration.yaml

```diff
@@ -3,10 +3,37 @@
 #The following properties must be defined for each instance types compute node configuration:
 #   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
 
+# For each supported cloud provider, the following properties must be provided for the machine type
+#   - cluster_wide_workload_machine_type: the instance type size to handle cluster wide workloads e.g ingress controllers, observability operators, etc 
+#   - kafka_workload_machine_type: the instance type size to handle the Kafka instance workload e.g brokers, kafka-admin API, etc
+
 ---
-developer:
-  compute_nodes_config:
-    max_compute_nodes: 3
-standard:
-  compute_nodes_config:
-    max_compute_nodes: 9
+# The version of OpenShift used during installation of newly auto created data plane clusters.
+# If the value is empty (default), then OpenShift Cluster Manager (OCM) will provision the data plane cluster using the latest version. 
+new_data_plane_openshift_version: ""
+# Whether to auto create a data plane cluster.
+# The value is used to control whether kas-fleet-manager (KFM) should trigger the creation of new data plane clusters.
+# If it is set to false, then KFM will only perform scale up evaluation without triggering scale up i.e a dry run of cluster's creation.
+# If set to true, then KFM will perform scale up evaluation and trigger scaling up if it is needed based on the evaluation results.
+enable_dynamic_data_plane_scale_up: false
+# Whether to auto delete a data plane cluster cluster
+# The value is used to control whether KFM should trigger the deletion of data plane clusters.
+# If it is set to false, then KFM will only perform scale down evaluation without triggering scale down i.e a dry run for clusters' deletion.
+# If set to true, then KFM will perform scale down evaluation and trigger scaling down if it is needed based on the evaluation results.
+enable_dynamic_data_plane_scale_down: false
+# machine type configuration per cloud provider 
+machine_type_per_cloud_provider:
+  aws:
+    cluster_wide_workload_machine_type: m5.2xlarge
+    kafka_workload_machine_type: r5.xlarge
```

Comment:
This needs to be splitted up onto:
```
standard_kafka_workload_xxx
developer_kafka_workload_xxx
```

Reply:
Is there a use-case already for it?

Reply:
:+1: . I am wondering if that belongs to dynamic scaling config or kafka instance types configuration

Reply:
I think there are some 'huge' overlaps between the dynamic scaling config (in the sense that it references cloud providers and instance type), the cloud providers config (it defines cloud providers and instance types limits), and the supported kafka instance types. 
One reason for keeping some references to cloud provider & instance type here (in the dynamic config) will that the info defined are dynamic scaling related, one doesn't have to think about defining them if they are in manual scaling mode. Thus the configuration info are contextualized. The downside is that, it makes for a bad experience when defining configuration as one has to hoops between 2/3 configuration files (especially if docs are missing ).

Reply:
We'll need to make the machine type configurable per instance type. 
The PR will be updated accordingly.

Reply:
Done, here is how it looks like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/708a1e134837be1dbb41061698652caa88ff514c/config/dynamic-scaling-configuration.yaml#L1-L59